docs/source/aws_https.rst

```diff
@@ -78,6 +78,28 @@ Example hitting API with IAM credentials:
         }
     }
 
+Example hitting API with the custom AWS credentials:
+
+.. code-block:: ts
+
+    // Call my helper function to get credentials with AWS.STS
+    const roleCredentials = await this.getAssumeRoleCredentials();
+
+    const awsCredentials = {
+        accessKey: roleCredentials.AccessKeyId,
+        secretKey: roleCredentials.SecretAccessKey,
+        sessionToken: roleCredentials.SessionToken,
+    };
+    const http = new AwsHttps(false, awsCredentials);
+
+    // Build request options from a method and URL
+    const url = new URL('https://www.onica.com/ping.json');
```

Comment:
Is there a URL that we can use that returns an HTTP 200?

Reply:
It isn't intended to actually be used - just a subtile promotion for Onica. I don't see an onica.com URL that returns JSON.